ptclient/cts/spi/dx/cl/ctMultiChangeDxa.vue

```diff
@@ -23,27 +23,22 @@
                 >
               </el-form-item>
             </el-form>
-            <DxaHistoryItem :dx="dx"></DxaHistoryItem>
+            <dxa-history-item :dx="dx" />
           </el-card>
         </el-col>
       </el-carousel-item>
     </el-carousel>
   </el-row>
 </template>
 <script>
-import DxaHistoryItem from './ctDxaHistory'
+import DxaHistoryItem from './ctDxaHistory.vue'
 import ormDx from '@/models/Dx'
 export default {
   components: {
     DxaHistoryItem,
   },
-  data() {
-    return {}
-  },
   computed: {
-    dxList() {
-      return ormDx.query().get()
-    },
+    dxList: () => ormDx.query().get(),
```

Comment:
There's no context here, an arrow function indicates this but also cleans up single line computed properties.